res/field/scripts/scripts_common.s

```diff
@@ -4,122 +4,122 @@
-    ScriptEntry _0494
-    ScriptEntry _00EC
-    ScriptEntry _05EA
-    ScriptEntry _0719
-    ScriptEntry _00EA
-    ScriptEntry _08FF
-    ScriptEntry _0901
-    ScriptEntry _093A
-    ScriptEntry _095C
-    ScriptEntry _0983
-    ScriptEntry _09F5
-    ScriptEntry _0BDD
-    ScriptEntry _0BEE
-    ScriptEntry _0FC3
-    ScriptEntry _0FCA
-    ScriptEntry _103A
-    ScriptEntry _0FA5
-    ScriptEntry _0FA7
-    ScriptEntry _1280
-    ScriptEntry _1282
-    ScriptEntry _12A8
-    ScriptEntry _12BA
-    ScriptEntry _12CD
-    ScriptEntry _12E0
-    ScriptEntry _12F3
-    ScriptEntry _1361
-    ScriptEntry _138C
-    ScriptEntry _139D
-    ScriptEntry _048B
-    ScriptEntry _13AB
-    ScriptEntry _09BD
-    ScriptEntry _1475
-    ScriptEntry _13BE
-    ScriptEntry _1477
-    ScriptEntry _1581
-    ScriptEntry _15D7
-    ScriptEntry _15B3
-    ScriptEntry _15D7
-    ScriptEntry _0A34
-    ScriptEntry _09CC
-    ScriptEntry _0910
-    ScriptEntry _0992
-    ScriptEntry _15BF
-    ScriptEntry _15D7
-    ScriptEntry _15E7
-    ScriptEntry _1636
-    ScriptEntry _164A
-    ScriptEntry _165E
-    ScriptEntry _16AA
-    ScriptEntry _15CB
-    ScriptEntry _15D7
-    ScriptEntry _170A
+    CommonScriptEntry HandleSignpostInput
+    CommonScriptEntry CheckBagPocketForItem
+    CommonScriptEntry Pokecenter_SpeakWithNurse
+    CommonScriptEntry _03E8
+    CommonScriptEntry _043B
+    CommonScriptEntry _0479
+    CommonScriptEntry _0494
+    CommonScriptEntry EmptyScript2
+    CommonScriptEntry _05EA
+    CommonScriptEntry ObtainPoketchApp
+    CommonScriptEntry EmptyScript1
+    CommonScriptEntry EmptyScript3
+    CommonScriptEntry SendToUndergroundPC
+    CommonScriptEntry ObtainUndergroundTrap
+    CommonScriptEntry ObtainUndergroundSphere
+    CommonScriptEntry ObtainAccessoryWaitForConfirm
+    CommonScriptEntry AddItemQuantityNoLineFeed
+    CommonScriptEntry BagIsFull
+    CommonScriptEntry _0BEE
+    CommonScriptEntry VendorGreetingGeneric
+    CommonScriptEntry _0FCA
+    CommonScriptEntry _103A
+    CommonScriptEntry EmptyScript4
+    CommonScriptEntry _0FA7
+    CommonScriptEntry EmptyScript5
+    CommonScriptEntry _1282
+    CommonScriptEntry _12A8
+    CommonScriptEntry _12BA
+    CommonScriptEntry _12CD
+    CommonScriptEntry _12E0
+    CommonScriptEntry _12F3
+    CommonScriptEntry _1361
+    CommonScriptEntry _138C
+    CommonScriptEntry _139D
+    CommonScriptEntry _048B
+    CommonScriptEntry _13AB
+    CommonScriptEntry ObtainContestBackdropWaitForConfirm
+    CommonScriptEntry EmptyScript6
+    CommonScriptEntry PrintPlateObtainedMessage
+    CommonScriptEntry _1477
+    CommonScriptEntry SetGenderBasedBGM
+    CommonScriptEntry FadeToDefaultMusic
+    CommonScriptEntry SetRivalBGM
+    CommonScriptEntry FadeToDefaultMusic
+    CommonScriptEntry AddItemQuantity
+    CommonScriptEntry ObtainContestBackdrop
+    CommonScriptEntry SendToUndergroundPCWithLinefeed
+    CommonScriptEntry ObtainAccessory
+    CommonScriptEntry SetTSURETEKEBGM
```

Comment:
Let's leave this unnamed for now since we definitely would not be using that BGM name.